.travis-build.sh

```diff
@@ -19,6 +19,16 @@ if [ $USER != "travis" ]; then
     exit 1
 fi
 
+if ! $TRAVIS_SECURE_ENV_VARS && [ $TRAVIS_BRANCH = "master" ]; then
+    printf '=%.0s' {1..70}
+    printf "\n       し(*･∀･)／   Thanks for the contribution!  ＼(･∀･*)ノ\n"
+    printf '=%.0s' {1..70}
+    printf "\n( ＾◡＾)っ Please submit your pull request against the develop branch.\n"
+    echo   "You can read the contribution guidelines at:"
+    echo   "https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org"
+    exit 1
+fi
+
```

Comment:
Except this would cause the build on master to fail would it not? Perhaps check if it's a pull request?